Cannot start nuxt:  Cannot find module '@nuxt/vite-builder'

### Environment

Nuxt CLI v3.0.0-rc.6                                                                                                                                                                               
RootDir: C:\Users\Ilya\Desktop\nuxt-app                                                                                                                                                            
Nuxt project info:                                                                                                                                                                                 

------------------------------
- Operating System: `Windows_NT`
- Node Version:     `v16.15.0`
- Nuxt Version:     `3.0.0-rc.6`
- Package Manager:  `yarn@3.2.1`
- Builder:          `vite`
- User Config:      `ssr`, `bridge`, `buildModules`, `css`, `windicss`, `build`
- Runtime Modules:  `-`
- Build Modules:    `nuxt-windicss@2.4.3`
------------------------------


### Reproduction

https://github.com/loremru/nuxt-bug

### Describe the bug

 ERROR  Cannot start nuxt:  Cannot find module '@nuxt/vite-builder'                                                                                                                                
Require stack:                                                                                                                                                                                             
- C:\Users\Ilya\Desktop\nuxt-app\index.js  

![image](https://user-images.githubusercontent.com/66596500/181235916-6dc16aca-2e2d-4acb-97f5-d51e5d4ba12e.png)


### Additional context

on majestic-ui version 0.0.83-13 its ok. But on latest versions there are nothing changed, only yarn version. Also latest versions works on clear vite.js

### Logs

_No response_

--- a/packages/nuxt/src/core/builder.ts
+++ b/packages/nuxt/src/core/builder.ts
@@ -61,7 +61,7 @@ function watch (nuxt: Nuxt) {
 async function bundle (nuxt: Nuxt) {
   try {
     const { bundle } = typeof nuxt.options.builder === 'string'
-      ? await importModule(nuxt.options.builder, { paths: [import.meta.url, nuxt.options.rootDir, nuxt.options.workspaceDir] })
+      ? await importModule(nuxt.options.builder, { paths: [nuxt.options.rootDir, nuxt.options.workspaceDir, import.meta.url] })
       : nuxt.options.builder
 
     return bundle(nuxt)

--- a/packages/nuxt/src/core/builder.ts
+++ b/packages/nuxt/src/core/builder.ts
@@ -61,7 +61,7 @@ function watch (nuxt: Nuxt) {
 async function bundle (nuxt: Nuxt) {
   try {
     const { bundle } = typeof nuxt.options.builder === 'string'
-      ? await importModule(nuxt.options.builder, { paths: nuxt.options.rootDir })
+      ? await importModule(nuxt.options.builder, { paths: [import.meta.url, nuxt.options.rootDir, nuxt.options.workspaceDir] })
       : nuxt.options.builder
 
     return bundle(nuxt)

